OpenRA.Mods.Common/Traits/Conditions/GrantChargedConditionOnToggle.cs

```diff
@@ -17,13 +17,13 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
-	[Desc("Allow deploying on specified charge to grant a condition for a specified duration.")]
-	public class GrantConditionOnDeployWithChargeInfo : PausableConditionalTraitInfo, IRulesetLoaded
+	[Desc("Grant a condition via player orders for a specified amount of time.")]
+	public class GrantChargedConditionOnToggleInfo : PausableConditionalTraitInfo, IRulesetLoaded
```

Comment:
This uses "Toggle" but most of the fields below use "Activat(ed|ion)". Can we standardise on Activation if this must have a suffix? (I still don't think it needs one.)

Reply:
we do use the name toggle for toggle actions

Reply:
🤷 
<img width="800" alt="Screenshot 2024-12-21 at 15 54 05" src="https://github.com/user-attachments/assets/1154c616-157a-4ffa-85a1-16e6dd51c85a" />

Reply:
ToggleState